scripts/find_unused_deps.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'toml-rb'
+require 'set'
+
+exit_code = 0
+
+Dir.glob("#{File.dirname(File.expand_path($PROGRAM_NAME))}/../**/*.toml").each do |file|
```

Comment:
This seems a bit complex, I'd say we can just `Dir.glob('**/*.toml')` and run it from the project's root.

Reply:
Fixed